guide/moderator/editassignedtags.md

```diff
@@ -0,0 +1,15 @@
+## User Handbook
+
+### Editing tags Assigned to a Thread
+
+Has a thread been assigned the wrong tags or do you need to add more? not to worry! this is easy to do.
```

Comment:
Sorry, really trying not to nitpick on prose, but sentences start with capital letters. "**N**ot to worry!" **T**his is easy to do."